src/main/scala/Main.scala

```diff
@@ -160,75 +113,32 @@ object Main {
   }
 
   // Function that removes a question from player's collection of questions
-  private def removeQuestionFromQuestionsOfPlayer(questionPicked:String, mapOfQuestions:Map[String, Character => Boolean]):Map[String, Character => Boolean]={
+  def removeQuestionFromQuestionsOfPlayer(questionPicked:String, mapOfQuestions:Map[String, Character => Boolean]):Map[String, Character => Boolean]={
     val updatedMapOfQuestions = mapOfQuestions.removed(questionPicked)
     updatedMapOfQuestions
   }
 
   // Functions that removes the characters with the specified criterion
-  private def removeCharacter(yesOrNo:String, listOfCharacters:List[Character], mapOfQuestions:Map[String, Character => Boolean], randomQuestion:String):List[Character] = {
+  def removeCharacter(yesOrNo:String, listOfCharacters:List[Character], mapOfQuestions:Map[String, Character => Boolean], randomQuestion:String):List[Character] = {
   val trueOrFalse:Boolean = if (yesOrNo=="yes"){
```

Comment:
// Simplify condition
```suggestion
  val trueOrFalse: Boolean = yesOrNo == "yes"
```